### DISCUSSION TITLE
Ruby'de (and ve &&) operatörleri arasında ki fark nedir ?

### DISCUSSION THREAD

**Discussion by BunyaminAkdeniz** [19/classroom#33] [Ruby]
![Ekran görüntüsü 2022-03-16 21-59-38](https://user-images.githubusercontent.com/91963771/158666770-8fc7b610-223b-45f1-b252-6ae59b982d45.png)

**Comment by roktas** [comment]
https://stackoverflow.com/questions/1426826/difference-between-and-and-in-ruby

TLDR;

- Operatör öncelikleri (precedence) arasında fark var
- `and` ve `or` kullanmayın (çok istisnai bazı shortcutlar da belki), daima `&&` ve `||` kullanın
